modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -17,44 +17,30 @@ Some administrators prefer to use static IP addresses so that each node's IP add
 ====
 endif::[]
 
-Preparing the bare metal node requires executing the following procedure from the provisioner node.
-
 .Procedure
 
-. Get the `oc` binary, if needed. It should already exist on the provisioner node.
-+
-[source,bash]
-----
-[kni@provisioner ~]$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/openshift-client-linux-$VERSION.tar.gz | tar zxvf - oc
-----
-+
-[source,bash]
-----
-[kni@provisioner ~]$ sudo cp oc /usr/local/bin
-----
+. Power off the bare metal node by using the baseboard management controller, and ensure it is off.
 
-. Power off the bare metal node via the baseboard management controller and ensure it is off.
-
-. Retrieve the user name and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the user name and password. In the following example, the user name is `root` and the password is `calvin`.
+. Retrieve the user name and password of the bare metal node's baseboard management controller. Then, create `base64` strings from the user name and password: 
```

Comment:
You should either make the second sentence a second step, or recast as: "...the bare metal node's baseboard management controller, and then, create `base64` strings from the user name and password:

Reply:
Updated as an additional step as suggested.